.github/workflows/publish.yaml

```diff
@@ -0,0 +1,58 @@
+name: Publish
+
+on:
+  push:
+    tags:
+    - '*'
+  workflow_dispatch:
+    inputs:
+      docker_tag_prefix:
+        description: "Enter the Docker Tag Prefix (e.g adhoc-test)"
+        required: true
+        type: string
+env:
+  ECR_REPOSITORY_URL: 673156464838.dkr.ecr.us-west-2.amazonaws.com
+  ECR_REPOSITORY_NAME: submission-report
```

Comment:
I suppose we need to create new repository for this one.

Reply:
https://github.com/MinaFoundation/gitops-infrastructure/pull/386 it done